machine_manager/README.rst

```diff
@@ -0,0 +1,27 @@
+Machine Manager
```

Comment:
You're right Daniel but the content of the modules are the same, just split in two parts. 
It's the price of the modularity required in OCA repositories: reuse low level components in miscellanous situations for high level solutions. Other users can share their opinions